CI/integration_tests/models/test_huggingface_flax_model_deployment.py

```diff
@@ -0,0 +1,142 @@
+"""
+ZnNL: A Zincwarecode package.
+
+License
+-------
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License v2.0 which accompanies this distribution, and is
+available at https://www.eclipse.org/legal/epl-v20.html
+
+SPDX-License-Identifier: EPL-2.0
+
+Copyright Contributors to the Zincwarecode Project.
+
+Contact Information
+-------------------
+email: zincwarecode@gmail.com
+github: https://github.com/zincware
+web: https://zincwarecode.com/
+
+Citation
+--------
+If you use this module please cite us with:
+
+Summary
+-------
+"""
+
+import os
+
+os.environ["CUDA_VISIBLE_DEVICES"] = "-1"
```

Comment:
Can we remove this one?

Reply:
Removed in [e4b04f5](https://github.com/zincware/ZnNL/pull/105/commits/e4b04f5a4b2da4b89e14d2f5fde28b499c962d76)